Taskfile.yml

```diff
@@ -9,6 +9,7 @@ vars:
   # Paths
   G_BUILD_DIR: "{{.ROOT_DIR}}/build"
   G_CORE_COMPONENT_BUILD_DIR: "{{.G_BUILD_DIR}}/core"
+  G_LATEST_NODEJS_DIR: "{{.G_BUILD_DIR}}/nodejs-latest"
```

Comment:
Would it be a better idea to keep the Node.js version fixed for both the linter and the `log-viewer-webui` project? While generally not expected, if there is any breaking API change after a new Node.js version is released, our automation pipelines will break as well.

Feel free to object.

Reply:
Sure